Doc/library/numbers.rst

```diff
@@ -10,7 +10,7 @@
 
 The :mod:`numbers` module (:pep:`3141`) defines a hierarchy of numeric
 :term:`abstract base classes <abstract base class>` which progressively define
-more operations.  None of the types defined in this module can be instantiated.
+more operations. None of the types defined in this module are intended to be instantiated.
```

Comment:
This isn't anything major, but you don't have to remove/reformat that whitespace. Officially, PEP 12 recommends 2 spaces after every sentence https://www.python.org/dev/peps/pep-0012/#general.

A lot of places in the docs don't follow this though. So I don't recommend trying to re-format other parts of the docs.
```suggestion
more operations.  None of the types defined in this module are intended to be instantiated.
```

Reply:
It wasn't intentional. My IDE did that on save. Will revert that.

Reply:
Yep no worries :). My IDE does that on C code too.